Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
Good point, and another reason to only run in Travis. I'll make that change tomorrow.